### PR TITLE
feat (DPLAN-3796) Fix bulk segement editor test

### PR DESCRIPTION
**Pull request by sierrazamudiodemos**
Description:
- Make work again SegmentBulkEditorServiceTest
- Rename file to SegmentBulkEditorServiceTest

### How to review/test
- Run the test in local, all tests should pass

Delete the checkbox if it doesn't apply/isn't necessary.

- [x] Tests updated/created
- [ ] Update documentation
- [ ] Link all relevant tickets
- [ ] Move the tickets on the board accordingly
- [ ] Data-Cy attributes added/updated ([conventions](https://dplan-documentation.demos-europe.eu/development/guidelines-conventions/coding-styleguides/twig_html.html#guideline-for-naming-cypress-hooks))

### CODE REVIEW

File: tests/backend/core/Core/Unit/Logic/Segment/RpcBulkEditor/SegmentBulkEditorServiceTest.php [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+        $user = $this->loginTestUser();
+        $assignee = $this->sut->detectAssignee($user->getId());
+        self::assertEquals($user->getId(), $assignee->getId());
+    }
+
+    public function testDetectInvalidIdAssignee(): void
+    {
+        self::expectException(UserNotFoundException::class);
+        $invalidAssigneeId = '123';
+        $this->sut->detectAssignee($invalidAssigneeId);
```
Comment (jalalifardemos) [inline tests/backend/core/Core/Unit/Logic/Segment/RpcBulkEditor/SegmentBulkEditorServiceTest.php:94]: We don't need any assert like assertNull here?

File: tests/backend/core/Core/Unit/Logic/Segment/RpcBulkEditor/SegmentBulkEditorServiceTest.php [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+        self::assertContains($this->segment1, $segments);
+        self::assertContains($this->segment2, $segments);
+    }
+
+    public function testGetInvalidSegments()
+    {
+        $invalidProcedureId = '123';
+        self::expectException(InvalidArgumentException::class);
+        $segments = $this->sut->getValidSegments([$this->segment1->getId(), $this->segment2->getId()], $invalidProcedureId);
```
Comment (jalalifardemos) [inline tests/backend/core/Core/Unit/Logic/Segment/RpcBulkEditor/SegmentBulkEditorServiceTest.php:115]: self::assertNotContains($this->segment1, $segments);
self::assertNotContains($this->segment2, $segments);

File: tests/backend/core/Core/Unit/Logic/Segment/RpcBulkEditor/SegmentBulkEditorServiceTest.php [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+        self::assertContains($tag2, $tags);
+    }
+
+    public function testGetInvalidTags(): void
+    {
+        $testTag1 = $this->getTagReference('testFixtureTag_1');
+        $testTag2 = $this->getTagReference('testFixtureTag_2');
+
+        self::expectException(InvalidArgumentException::class);
+        $tags = $this->sut->getValidTags([$testTag1, $testTag2], $this->procedure->getId());
```
Comment (jalalifardemos) [inline tests/backend/core/Core/Unit/Logic/Segment/RpcBulkEditor/SegmentBulkEditorServiceTest.php:134]: self::assertNotContains ?

**Comment by jalalifardemos** [comment]
Not block, just some offers

File: tests/backend/core/Core/Unit/Logic/Segment/RpcBulkEditor/SegmentBulkEditorServiceTest.php (hunk 1 above)
Comment (sierrazamudiodemos) [reply tests/backend/core/Core/Unit/Logic/Segment/RpcBulkEditor/SegmentBulkEditorServiceTest.php:94]: done bb2fefc7b61d51f164aeac8f977a779a7f312917

File: tests/backend/core/Core/Unit/Logic/Segment/RpcBulkEditor/SegmentBulkEditorServiceTest.php (hunk 2 above)
Comment (sierrazamudiodemos) [reply tests/backend/core/Core/Unit/Logic/Segment/RpcBulkEditor/SegmentBulkEditorServiceTest.php:115]: done bb2fefc7b61d51f164aeac8f977a779a7f312917

**Review by jalalifardemos — APPROVED** [review]
Thanks

File: tests/backend/core/Core/Unit/Logic/Segment/RpcBulkEditor/SegmentBulkEditorServiceTest.php (hunk 3 above)
Comment (sierrazamudiodemos) [reply tests/backend/core/Core/Unit/Logic/Segment/RpcBulkEditor/SegmentBulkEditorServiceTest.php:136]: done bb2fefc7b61d51f164aeac8f977a779a7f312917

**Comment by sierrazamudiodemos** [comment]
Thanks to you for the comments

**Pull request merged**